### DISCUSSION TITLE
How to upload Valheim Dedicated Server Save?

### DISCUSSION THREAD

**Discussion by rock628** [GameServerManagers/LinuxGSM#3249] [Q&A] [answered]
Hallo,

I want upload my Valheim Dedicated Server(On steam file save) overwrite my Google Cloud VM save

but I don't know how to do that, please help and thanks a lot.

**Comment by jkavenaugh** [comment, marked as answer]
I copied my world to .config/unity3d/IronGate/Valheim/worlds/
The windows location was AppData\LocalLow\IronGate\Valheim\worlds

The map is the same seed. Structures that we built are missing.

**Reply by rock628** [reply]
thanks a lot
